README.md

```diff
@@ -0,0 +1,53 @@
+JW Platform API Client
```

Comment:
May be worth making `make run` explicit in the `README`

Reply:
Going to leave it out for now